src/RestServer/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "PluginConfiguration": {
+    "Network": 860833102,
+    "BindAddress": "127.0.0.1",
+    "Port": 10339,
+    "KeepAliveTimeout": 120,
+    "SslCertFile": "",
+    "SslCertPassword": "",
+    "TrustedAuthorities": [], // Example: "aec13cdd5ea6a3998aec14ac331ad96bedbb770f" (Thumbprint)
+    "EnableBasicAuthentication": false,
+    "RestUser": "",
+    "RestPass": "",
+    "EnableCors": true,
+    "AllowOrigins": [], // Example: "http://www.example.com"
+    "DisableControllers": [],
+    "EnableCompression": true,
+    "CompressionLevel": "SmallestSize", // "Fastest" or "Optimal" or "NoCompression" or "SmallestSize"
```

Comment:
No comments are allowed by JSON specification. If some comments are needed, then they can be mentioned in the markdown documentation.

Reply:
> No comments are allowed by JSON specification. If some comments are needed, then they can be mentioned in the markdown documentation.

This is not necessary. Use it in neo core already.

Reply:
Microsoft `dotnet` allow comments.